github/repos_hooks.go

```diff
@@ -92,6 +92,7 @@ func (h Hook) String() string {
 // information.
 type createHookRequest struct {
 	// Config is required.
+	Name   string                 `json:"name,omitempty"`
```

Comment:
Please remove the `,omitempty` since it is not a pointer and with this PR, it will always be defined.

Reply:
Removed as per request.